setup.py

```diff
@@ -19,33 +17,32 @@
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
-
-def read(*names, **kwargs):
-    return io.open(
-        join(dirname(__file__), *names),
-        encoding=kwargs.get('encoding', 'utf8')
-    ).read()
-
 requirements = [
-    read('requirements.txt')
+    'python-dateutil==2.6.1',
+    'rasterio==1.0a12',
+    'Click>=6.0',
+    'geojson==2.3.0',
+    'requests==2.18.0',
+    'typing==3.6.2;python_version<"3.5"'
 ]
 
 setup_requirements = [
-    # TODO(notthatbreezy): put setup requirements (distutils extensions, etc.) here
+    # NOQA TODO(notthatbreezy): put setup requirements (distutils extensions, etc.) here
 ]
 
 test_requirements = [
-    # TODO: put package test requirements here
+    # NOQA TODO: put package test requirements here
 ]
 
 setup(
     name='pystac',
-    version='0.1.0',
-    description="Python command line utilities for interacting with and creating STAC compliant files.",
+    version='0.1.1',
+    description=("Python command line utilities for interacting with and"
+                 "creating STAC compliant files."),
     long_description=readme + '\n\n' + history,
     author="Raster Foundry",
     author_email='info@rasterfoundry.com',
-    url='https://github.com/notthatbreezy/pystac',
+    url='https://github.com/rasterfoundry/pystac',
     packages=find_packages(),
     py_modules=[splitext(basename(path))[0] for path in glob('pystac/*.py')],
     entry_points={
```

Comment:
Couldn't comment on the appropriate line, but I think that it would be good to ask in the channel for some appropriate `keywords` before publishing.

Reply:
As far as keywords go, not sure really what would be good, maybe the following:
 - imagery
 - raster
 - catalog

At least as a start.